modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee

```diff
@@ -951,21 +950,24 @@ class FileRowSummaryController extends Backbone.View
 			mm = '0' + mm
 		today = '_' + mm + '_' + dd + '_' + yyyy
 
-		# Rename the SDF file representing the current
-		currentFileName = originalFileName.replace "\.sdf", today + "_current_state.sdf"
-    
+		#In some cases, there are artifact .sdf extensions in a file name that cause downloading issues when renanubg a file to .zip
+		#we need to replace ONLY the last .sdf in the file with ".zip" or "{current_date}_current_state.sdf", so we can't use replace which will replace the first
```

Comment:
Should I add logic here to check if the last four characters are actually ".sdf"? I don't think there would be a case where it wouldn't be an .sdf file - but if it is not it would break? I feel like blindly removing the last four characters may not be the best option.

Reply:
I agree this seems a little hacky, but we truly aren't expecting any cases where the file isn't .sdf, as that's the only format we support. I think it's OK as is.